packages/salesforcedx-vscode-replay-debugger/src/states/frameEntryState.ts

```diff
@@ -26,7 +26,7 @@ export class FrameEntryState extends FrameState implements DebugLogState {
           logContext.getFrames().length,
           this._frameName,
           sourceUri
-            ? new Source(basename(sourceUri), Uri.parse(sourceUri).path)
+            ? new Source(basename(sourceUri), Uri.parse(sourceUri).fsPath)
```

Comment:
This is probably the key. Using `fsPath` will take care of the encoding for you from the VS Code side.